cfme/cloud/availability_zone.py

```diff
@@ -32,7 +33,7 @@ class AvailabilityZoneDetailsToolBar(View):
     view_selector = View.nested(ItemsToolBarViewSelector)
 
 
-class AvailabilityZoneEntities(View):
+class AvailabilityZoneEntities(BaseEntitiesView):
     """View containing the widgets for the main content pane"""
     title = Text('//div[@id="main-content"]//h1')
     table = Table("//div[@id='gtl_div']//table")
```

Comment:
I think you should likely expand the scope of the refactor a bit here.

If using BaseEntitiesView, there shouldn't be a need for these table or title widgets (unless they're unique).

This means re-writing the navigation step below on L138 to use the entity functions and used `surf_pages`.